html-tables/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Tables</title>
+  <style>
+    th {
+      font-weight: bold;
+    }
+
+  </style>
+</head>
+<body>
+  <table>
+    <h1>Student Grade Table</h1>
+
+  <h1>Student Grade Table</h1><tr>
```

Comment:
What happened here? You should move the `h1` out of the `table` element (and only have one, not two)